lib/checks/mobile/css-orientation-lock-evaluate.js

```diff
@@ -103,21 +103,31 @@ function cssOrientationLockEvaluate(node, options, virtualNode, context) {
 
     const transformStyle =
       style.transform || style.webkitTransform || style.msTransform || false;
-    if (!transformStyle) {
+    if (!transformStyle && !style.rotate) {
       return false;
     }
 
+    let transformFn;
+    let transformFnValue;
```

Comment:
Nitpick; I suggest pulling this code out into its own function.

```suggestion
    const { transformType, transformValue } = getCssTransform(style);
```